source/hic_hal/nxp/lpc11u35/uart.c

```diff
@@ -99,6 +99,9 @@ int32_t uart_set_configuration(UART_Configuration *config)
     baudrate = config->Baudrate;
     // Compute baud rate dividers
     mv = 15;
+#ifdef BOARD_RTL8195AM
+    config->Baudrate = 38400;
```

Comment:
It's confirmed a hardware bug. The main processor can only modulate 38400 baudrate with DAP UART.  It shall be fixed in next generation, but for RTL8195AM, the baudrate has to be fixed to 38400.